app/src/main/java/com/kickstarter/libs/FeatureKey.java

```diff
@@ -3,4 +3,5 @@
 public final class FeatureKey {
   private FeatureKey() {}
 
+  public static final String ENABLE_HORIZONTAL_REWARDS = "enable_horizontal_rewards";
```

Comment:
A feature key is usually the name of a preference, some examples of other feature keys we've had are 
```  public static final String ANDROID_MESSAGES = "android_messages";```
```public static final String ANDROID_SURVEYS = "android_surveys";```

So I think this should have a more descriptive name like `internal_horizontal_rewards`.

Reply:
Fixed!